homeassistant/components/tplink/__init__.py

```diff
@@ -189,7 +189,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     parent_coordinator = TPLinkDataUpdateCoordinator(hass, device, timedelta(seconds=5))
     child_coordinators: list[TPLinkDataUpdateCoordinator] = []
 
-    if device.is_strip:
+    if device.children:
```

Comment:
We need to check for IotStrip here. The only reason for these extra coordinators is that the HS300 allows a limited number of concurrent request, and fetching the emeter information requires separate ones.

Reply:
That's done